spec/features/admins.feature

```diff
@@ -13,6 +13,11 @@ Feature: Admin
     And I can add a new resource
     And I can see that the resource has been added
     And I can see a bookings of a resource and all the info of them
+    And I can cancel or reallocate a booking
+    And I cannot see a disabled resource to reallocate a booking
```

Comment:
"I cannot reallocate the booking to a disabled resource"

Y falta el "I cannot reallocate the booking to a resource with an overlapped booking."

Además, quizás estaría bien renombrar booking como event.